modules/adding-node-iso-configs.adoc

```diff
@@ -0,0 +1,154 @@
+== YAML file parameters
+
+Configuration parameters that can be specified in the `nodes-config.yaml` file are described in the following table:
+
+[cols=".^4l,.^3,.^2",options="header"]
+|====
+|Parameter|Description|Values
+
+|hosts:
+|Host configuration.
+|An array of host configuration objects.
+
+|hosts:
+  hostname:
+|Hostname.
+Overrides the hostname obtained from either the Dynamic Host Configuration Protocol (DHCP) or a reverse DNS lookup.
+Each host must have a unique hostname supplied by one of these methods, although configuring a hostname through this parameter is optional.
+|String.
+
+|hosts:
+  interfaces:
+|Provides a table of the name and MAC address mappings for the interfaces on the host.
+If a `NetworkConfig` section is provided in the `nodes-config.yaml` file, this table must be included and the values must match the mappings provided in the `NetworkConfig` section.
+|An array of host configuration objects.
+
+|hosts:
+  interfaces:
+    name:
+|The name of an interface on the host.
+|String.
+
+|hosts:
+  interfaces:
+    macAddress:
+|The MAC address of an interface on the host.
+|A MAC address such as the following example: `00-B0-D0-63-C2-26`.
+
+|hosts:
+  role:
+|Defines whether the host is a `master` or `worker` node.
+Currently, only worker nodes can be added using this process.
+|`worker`.
+
+|hosts:
+  rootDeviceHints:
+|Enables provisioning of the {op-system-first} image to a particular device.
+The node-adding tool examines the devices in the order it discovers them, and compares the discovered values with the hint values.
+It uses the first discovered device that matches the hint value.
+|A dictionary of key-value pairs.
+For more information, see "Root device hints" in the "Setting up the environment for an OpenShift installation" page.
+
+|hosts:
+  rootDeviceHints:
+    deviceName:
+|The name of the device the {op-system} image is provisioned to.
+|String.
+
+|hosts:
+  networkConfig:
+|The host network definition.
+The configuration must match the Host Network Management API defined in the link:https://nmstate.io/[nmstate documentation].
+|A dictionary of host network configuration objects.
+
+|====
+
+
+[id="adding-node-iso-flags-config_{context}"]
+== Command flag options
+
+Command flags for the `oc adm node-image create` command are described in the following table:
+
+[cols=".^4,.^3,.^2",options="header"]
+|====
+|Flag|Description|Values
+
+|`--certificate-authority`
+|The path to a certificate authority bundle to use when communicating with the managed container image registries.
+If the `--insecure` flag is used, this flag will be ignored.
+|String
+
+|`-c`, `--cpu-architecture`
+|The CPU architecture to be used to install the node.
+This flag can be used to create only a single node, and the `--mac-address` flag must be defined.
+|String
+
+|`--dir`
+|The path containing the configuration file, if provided.
+This path is also used to store the generated artifacts.
+|String
+
+|`--hostname`
```

Comment:
Could be possible to group toghether the single node flags (or to distinguish them from the other flags)? In the console output is not possible, but I'd prefer in the documentation to avoid the confusion of interleaving different kind of flags